uint32 num_frames
---
float64 average_visible_area
